lib/system.rb

```diff
@@ -74,7 +74,7 @@ def create_archive(file_list, archive, exclude = [])
       "The following files are packaged in #{archive}: " + Array(file_list).join(", ")
     )
     created = !File.exist?(archive)
-    out = File.open(archive, "w")
+    out = File.open(archive, "wb")
```

Comment:
the ruby docs are not particularly helpful on this (https://ruby-doc.org/core-2.7.2/File.html  doesnt describe the modes available) , I assume thats writing as binary or something similar?

Reply:
oops, your IO link does have it
```
"b"  Binary file mode
     Suppresses EOL <-> CRLF conversion on Windows. And
     sets external encoding to ASCII-8BIT unless explicitly
     specified.
```

Reply:
https://ruby-doc.org/core-2.7.2/IO.html#method-c-new-label-IO+Open+Mode